### PR TITLE
Created dcaff_shovel for april fools

### PR DESCRIPTION
**Pull request by Thunderpx11**
Hey merg,

if ya ever wanna add this item, here it is :)

-Thunderpx_11

### CODE REVIEW

File: dcaff_shovel_for_april_fools.dsc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#needs custom texturepack for the custom.fart_reverb
+#dcaff approved
+dcaff95_shovel:
+    type: item
```
Comment (mergu) [inline dcaff_shovel_for_april_fools.dsc:4]: although the denizen guides use 4 space indents, i very strongly prefer 2 space indents, would recommend updating your vscode settings to use them and swapping this file to use 2 space indents - https://www.kindacode.com/article/vs-code-how-to-change-indentation-2-spaces-4-spaces/

File: dcaff_shovel_for_april_fools.dsc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            input:
+                - dirt|dirt|dirt
+                - dirt|netherite_shovel|dirt
+                - dirt|dirt|dirt
+
+fart_Reverb:
+    type: world
+    events:
+        on player breaks block with:dcaff95_shovel:
+        - playsound <player> sound:custom.fart_reverb custom
```
Comment (mergu) [inline dcaff_shovel_for_april_fools.dsc:26]: no newline @ end of file,
see this guide for how to get vscode to do it automatically - https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

File: dcaff_shovel_for_april_fools.dsc [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    type: item
+    debug: false
+    material: netherite_shovel
+    data:
+        discover: true
+    display name: Dcaff's shovel
+    enchantments:
+        - unbreaking: 3
+    lore:
+        - "The ground shakes from these unheavenly sounds."
```
Comment (mergu) [inline dcaff_shovel_for_april_fools.dsc:13]: make sure to correct any underlined issues the denizen extension highlights in vscode
image

File: dcaff_shovel_for_april_fools.dsc [hunk 4]
```diff
@@ -0,0 +1,26 @@
+#needs custom texturepack for the custom.fart_reverb
+#dcaff approved
```
Comment (mergu) [inline dcaff_shovel_for_april_fools.dsc:2]: should be a space between `#` and the comment, not an error but is a pretty common code style requirement across the industry

**Review by mergu — COMMENTED** [review]
hey man, congrats on your first PR! had some comments on how to improve this. looking forward to what else you create

File: dcaff_shovel_for_april_fools.dsc (hunk 4 above)
Comment (Thunderpx11) [reply dcaff_shovel_for_april_fools.dsc:2]: ok, will update that when i get the time xD thanks!

Comment (Thunderpx11) [reply dcaff_shovel_for_april_fools.dsc]: done

**Pull request closed without merging**